tests/test_plot_time_series.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import pandas as pd
+import numpy as np
+from plot_time_series import plot_time_series
+
+def create_test_data():
+    dates = pd.date_range('2020-01-01', periods=10, freq='D')
+    return pd.DataFrame({
+        'date': dates,
+        'sales': range(10),
+        'expenses': range(10, 20)
+    })
+
+def test_valid_input():
```

Comment:
For testing the validity of an input, I think it would be better if you could raise an error in the function and then test whether the function raises the correct error here in the testing file. This is also more clear for the users to know what exactly the problem is if they accidentally entered an invalid argument.